recipes/acado/all/conanfile.py

```diff
@@ -93,6 +93,8 @@ def package(self):
         tools.remove_files_by_mask(self.package_folder, "*.pdb")
 
     def package_info(self):
+        self.env_info.ACADO_TEMPLATE_PATHS = os.path.join(self.package_folder, "include", "acado", "code_generation", "templates")
```

Comment:
Imho, it's nice to emit info that some import variable is set.
```suggestion
        acado_template_paths = os.path.join(self.package_folder, "include", "acado", "code_generation", "templates")
        self.output.info("Setting ACADO_TEMPLATE_PATHS environment variable: {}".format(acado_template_paths))
        self.env_info.ACADO_TEMPLATE_PATHS = acado_template_paths
```

Reply:
thanks, done